### PR TITLE
Dm 374 as a c8y user i should ensure that the documentation created by the earlier user stories are integrated and pushed to the develop

### PR DESCRIPTION
**Pull request by atulpandasag**

### CODE REVIEW

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 1]
```diff
@@ -11,3 +11,9 @@ Release 10.15.0 includes the following new features or major feature enhancement
 In an effort to provide you a more consistent experience across the platform the **All devices** page has been re-shaped. It is now implemented in the data grid fashion you already are familiar with from **Subassets** and **Search** pages. This offers improved appearence in various viewport sizes, easier filtering and sorting, pagination option (only for users with global inventory access) and sets solid ground for further improvements.
 
 ![All devices](/images/release-notes/devmgmt-devices-alldevices.png)
+
+### Multiple LNS Connection for Sigfox and Actility
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:15]:
```suggestion
### Multiple LNS connection for Sigfox and Actility
```

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 2]
```diff
@@ -11,3 +11,9 @@ Release 10.15.0 includes the following new features or major feature enhancement
...
+
+The Sigfox and Actility agent now supports configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:17]:
```suggestion
The Sigfox and Actility agents now support the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
```

File: content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md [hunk 3]
```diff
@@ -208,4 +209,21 @@ Considering the pool of connections to microservices has a limited size (default
 MTM-34911</td>
 </tr>
 
-</tbody></table></div>
+<tr>
```
Comment (BeateRixen) [inline content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md:212]: @atulpandasag The regular release notes entries are handled by the release notes fields in the Jira ticket and will be added by me from there. I will leave them here now, but in the future you only have to fill in the fields in the tickets.

File: content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md [hunk 4]
```diff
@@ -208,4 +209,21 @@ Considering the pool of connections to microservices has a limited size (default
...
+<td>Sigfox</td>
+<td>The Sigfox agent earlier used to support only one connectivity setting per tenant.
```
Comment (BeateRixen) [inline content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md:214]:
```suggestion
<td>The Sigfox agent earlier used to support only one connectivity setting per tenant. 
```

File: content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md [hunk 5]
```diff
@@ -7,19 +7,27 @@ layout: bundle
...
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
 </thead><tbody>
 
+
+<tr>
+<td>Actility</td>
```
Comment (BeateRixen) [inline content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md:25]: If you add entries manually here, please make sure that they are sorted correctly (alphabetically). Moved the entry for Actility accordingly. Moreover I moved both entries to the Improvements section as they are not a bug fix.

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 6]
```diff
@@ -11,3 +11,9 @@ Release 10.15.0 includes the following new features or major feature enhancement
 In an effort to provide you a more consistent experience across the platform the **All devices** page has been re-shaped. It is now implemented in the data grid fashion you already are familiar with from **Subassets** and **Search** pages. This offers improved appearence in various viewport sizes, easier filtering and sorting, pagination option (only for users with global inventory access) and sets solid ground for further improvements.
 
 ![All devices](/images/release-notes/devmgmt-devices-alldevices.png)
+
+### Multiple LNS connection for Sigfox and Actility
+
+The Sigfox and Actility agents now support the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
+
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:18]: Please add a link to the documentation: "For details, refer to []()".

File: content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md (hunk 3 above)
Comment (atulpandasag) [reply content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md]: Sure. Thank you. I'll take care of it.

File: content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md (hunk 5 above)
Comment (atulpandasag) [reply content/release-10-15-0/devices-10-15-0-bundle/10-15-0-0.md:25]: Sure, Thank you.

File: content/release-10-15-0/whatsnew-10-15-0.md (hunk 6 above)
Comment (atulpandasag) [reply content/release-10-15-0/whatsnew-10-15-0.md:22]: Done.

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 7]
```diff
@@ -11,3 +11,13 @@ Release 10.15.0 includes the following new features or major feature enhancement
 In an effort to provide you a more consistent experience across the platform the **All devices** page has been re-shaped. It is now implemented in the data grid fashion you already are familiar with from **Subassets** and **Search** pages. This offers improved appearence in various viewport sizes, easier filtering and sorting, pagination option (only for users with global inventory access) and sets solid ground for further improvements.
 
 ![All devices](/images/release-notes/devmgmt-devices-alldevices.png)
+
+### Multiple LNS connection for Sigfox and Actility
+
+The Sigfox and Actility agents now support the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
+<br>
+For details, refer to
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:19]:
```suggestion
For details, refer to [Sigfox](https://cumulocity.com/guides/protocol-integration/sigfox) or [Actility LoRa](https://cumulocity.com/guides/protocol-integration/lora-actility) in the *Protocol integration guide*.
```

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 8]
```diff
@@ -11,3 +11,13 @@ Release 10.15.0 includes the following new features or major feature enhancement
...
+<br> Sigfox : https://cumulocity.com/guides/protocol-integration/sigfox
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:20]:
```suggestion
```

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 9]
```diff
@@ -11,3 +11,13 @@ Release 10.15.0 includes the following new features or major feature enhancement
...
+<br> Actility : https://cumulocity.com/guides/protocol-integration/lora-actility
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:21]:
```suggestion
```

**Review by BeateRixen — APPROVED** [review]

**Review by miryalarahul — APPROVED** [review]

**Pull request merged**